channels/generic/websocket.py

```diff
@@ -9,12 +11,15 @@ class WebsocketConsumer(SyncConsumer):
     Base WebSocket consumer. Provides a general encapsulation for the
     WebSocket handling model that other applications can build on.
     """
+    groups = []
```

Comment:
would it not be better to have groups as a `Set[str]`? we do not want to have repeats correct?

Reply:
We're not defining what type it is, the code just wants an iterable provided by the user. No point putting `set()` here rather than `[]`.